src/cmd/linuxkit/run_packet.go

```diff
@@ -267,3 +302,28 @@ func sshHostKey(host string) (ssh.PublicKey, error) {
 	}
 	return hostKey, nil
 }
+
+// This implements a http.FileSystem which only responds to specific files.
+type serveFiles struct {
+	files []string
+}
+
+func (fs serveFiles) Open(name string) (http.File, error) {
+	name = strings.TrimLeft(name, "/")
```

Comment:
not this part, but some other part, I think. URL paths are always `/` but the filename passed in might be windows so the `name == fn` may fail